packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -108,21 +92,17 @@ const getBundleAnalyzerPlugin = (name = 'report', pluginOptions) =>
 
 const entryPointExists = (segments) => {
     for (let ext of ['.js', '.jsx', '.ts', '.tsx']) {
```

Comment:
nit: should we use `SUPPORTED_FILE_EXTENSIONS` here as well?

```suggestion
    for (let ext of SUPPORTED_FILE_EXTENSIONS) {
```